tests/expected/unittest.jl

```diff
@@ -0,0 +1,20 @@
+import unittest
```

Comment:
when I try running the .jl locally I get
```
Traceback (most recent call last):
  File "/home/jayvdb/python/py2many/tests/test_cli.py", line 305, in test_generated
    assert (
AssertionError: Execution of unit_test.jl failed:
b''b'ERROR: LoadError: ArgumentError: Package unittest not found in current path.\n- Run `import Pkg; Pkg.add("unittest")` to install the unittest package.\nStacktrace:\n [1] macro expansion\n   @ ./loading.jl:1630 [inlined]\n [2] macro expansion\n   @ ./lock.jl:267 [inlined]\n [3] require(into::Module, mod::Symbol)\n   @ Base ./loading.jl:1611\nin expression starting at /home/jayvdb/python/py2many/tests/build/unit_test.jl:1\n'
```
However I cant find a Julia Package called unittest, or much Julia code on Github which uses ["assertEqual"](https://github.com/search?q=assertEqual+language%3Ajulia&type=code) .

Reply:
This would require a special translation that I have implemented in my fork. But I will maybe try to split it up to avoid introducing too many changes at once.